### PR TITLE
TL translations for import-custom.asc - 1,269 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
Please feel free to review my work. Thank you, have a great day!

### CODE REVIEW

**Comment by heywens** [comment]
Hi @josteinfrancois, I have checked your translation and found no words/phrases that require updates. Thank you for ensuring quality in your contribution :)

File: book/09-git-and-other-scms/sections/import-custom.asc [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-If none of these tools works for you, you have a more obscure tool, or you otherwise need a more custom importing process, you should use `git fast-import`.
-This command reads simple instructions from stdin to write specific Git data.
-It's much easier to create Git objects this way than to run the raw Git commands or try to write the raw objects (see <<_git_internals#_git_internals>> for more information).
-This way, you can write an import script that reads the necessary information out of the system you're importing from and prints straightforward instructions to stdout.
-You can then run this program and pipe its output through `git fast-import`.
+(((git na mga utos, fast-import)))
+(((Pag-import, mula sa mga iba)))
+Kung iyong sistema ay hindi isa sa nasa itaas, kailangan mong maghanap ng isang importer sa online - ang kalidad na mga importer ay maaaring magamit para sa maraming ibang mga sistema, nagsasama ng CVS, Clear Case, Visual Source Safe, kahit isang direktoryo ng mga archive.
```
Comment (severinolorillajr) [inline book/09-git-and-other-scms/sections/import-custom.asc]: Kung ang iyong sistema

Comment (severinolorillajr) [reply book/09-git-and-other-scms/sections/import-custom.asc]: kalakip na ang CVS, Clear Case

**Comment by josteinfrancois** [comment]
Thank you for your comments @heywens  and @severinolorillajr . Have a great day.

**Pull request merged**